crypto/ocsp/ocsp_vfy.c

```diff
@@ -84,6 +85,12 @@ int OCSP_basic_verify(OCSP_BASICRESP *bs, STACK_OF(X509) *certs,
             OCSPerr(OCSP_F_OCSP_BASIC_VERIFY, ERR_R_X509_LIB);
             goto f_err;
         }
+        if ((flags & OCSP_PARTIAL_CHAIN) != 0
+                && (vp = X509_STORE_CTX_get0_param(ctx)) != NULL) {
+            unsigned long f = X509_VERIFY_PARAM_get_flags(vp);
+
+            X509_VERIFY_PARAM_set_flags(vp, f | X509_V_FLAG_PARTIAL_CHAIN);
```

Comment:
I note that `X509_VERIFY_PARAM_set_flags()` does the "OR-ing in" part for us (it could perhaps be called "add flags" not "set flags"), so the preceding `X509_VERIFY_PARAM_get_flags()` is redundant and gives a confusing impression to the reader of the code.

Reply:
This already got approved, so unless @ddvo says he'll re-review I am not going to make the change.  If nothing else, the code shows how badly-named the API is, since it fooled two seasoned developers. :(

Reply:
Since Akamai decided to make the change, I propagated it here.  I rebased and squashed and asked @ddvo to re-review.  It would be nice if this made it into the BETA, but if not it's on your head, @kaduk :)